.cirrus.yml

```diff
@@ -808,7 +812,9 @@ release_task:
 release_test_task:
     name: "Optional Release Test"
     alias: release_test
-    only_if: $CIRRUS_PR != ''
+    # Allow running manually as part of release PR preperation
+    # see RELEASE_PROCESS.md
+    only_if: *not_build
```

Comment:
I'm confused about this change too. Won't this make it so this step starts running on PRs, which would be Very Bad?

Reply:
This task only happens when manually triggered.  The only point of including an `only_if` is to prevent the task from even showing up in `[CI:DOCS]` or `[CI:BUILD]` (previously it showed up for every PR and branch-build).

Reply:
I fixed up the comments here, should be more clear now.